tests/Microsoft.Identity.Test.Common/Core/Helpers/TraceLogger.cs

```diff
@@ -0,0 +1,98 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Test.Common.Core.Helpers
+{
+    internal class TraceLogger : ICoreLogger
+    {
+        private readonly string _prefix;
+
+        public TraceLogger(string prefix)
+        {
+            _prefix = prefix;
+        }
+
+        public Guid CorrelationId => Guid.Empty;
+
+        public string ClientName => "MSAL.Test";
+
+        public string ClientVersion => "0";
+
+        public bool PiiLoggingEnabled => true;
```

Comment:
Is this to test Pii logging with our test accounts?

Reply:
Normally we use the LogCallback to do logging, but some of our internal objects, use the actual ICoreLogger object. This is just a logger that logs via Trace.WriteLine (which end up in the test log)